public/js/actions/subscriptions.js

```diff
@@ -33,6 +33,39 @@ export function getUserSubscriptions(fetch=api.fetch) {
 }
 
 
+export function getFilteredUserSubscriptions(payMethodUri, fetch=api.fetch) {
+  // TODO This should be a specific API request
+  // see https://github.com/mozilla/payments-service/issues/127
+  return dispatch => {
+
+    dispatch({
+      type: actionTypes.LOADING_FILTERED_USER_SUBS,
+    });
+
+    fetch({
+      method: 'get',
+      url: '/braintree/subscriptions/',
+      context: this,
+    }).then(data => {
+      console.log('got subscriptions from API:', data);
+      var filteredSubs = data.subscriptions.filter(function(item) {
+        return item.paymethod === payMethodUri;
+      });
+      dispatch({
+        type: actionTypes.GOT_FILTERED_USER_SUBS,
```

Comment:
I was thinking I'll change this function and the const to something more like `getSubsByPayMethod` and `GOT_SUBS_BY_PAY_METHOD`